src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php

```diff
@@ -59,7 +59,12 @@ public function testOpeningDisallowedPaths($path, $isAllowed)
             ->disableOriginalConstructor()
             ->getMock();
 
-        $controller = new ProfilerController($urlGenerator, $profiler, $twig, array(), 'bottom', null, __DIR__.'/../..');
+        if ($withSymfonyBlacklist) {
+            $blacklistRegexp = "'(?:/\.|(?<!\.php|\.twig)$|ProjectContainer\.php$|/get[^/]+Service\.php$|^/(?:app/(?!Resources/(?:[^/]++/)?views/)|config/|var/))'i";
```

Comment:
You're not testing the regex that is actually configured but a copy of it. So, if the effective regex is changed, this test won't catch it.

Reply:
agreed, fixed